esp/src/eclwatch/SearchResultsWidget.js

```diff
@@ -43,7 +43,7 @@ define([
             lang.mixin(this.params, {
                 searchText: searchText
             });
-            this.searchText = searchText;
+            this.searchText = searchText.toUpperCase();
```

Comment:
With this change - none of the other ones are now needed (comparing with your closed PR)?